final_prep/mod_zero_hero.rb

```diff
@@ -56,29 +113,90 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
+scary_monster = {
+  'name' => 'Really Big Foot',
+  'smell' => 'mold',
+  'weight' => 540,
+  'citiesDestroyed' => ['Jericho', 'Persepolis', 'Aphek'],
+  'luckyNumbers' => [13, 96, 800],
+  'addrees' => {
+    'number' => 23,
+    'street' => 'Gremlin Ave',
+    'state' => 'New Jersey',
+    'zip' => 12666
+  }
+}
+puts scary_monster
+
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
+
+class SuperHero
+  attr_accessor :name, :super_power, :age
+  def initialize(name, super_power, age)
+    @name = name
+    @super_power = super_power
+    @age = age
+    
+    @@arch_nemesis = "The Syntax Error"
+    @@power_level = 100
+    @@energy_level = 50
+  end
+
+
 
 # - Create the following class methods
 #   - say_name, should print the hero's name to the terminal
 #   - maximize_energy, should update the energy_level to 1000
 #   - gain_power, should take an argument of a number and INCREASE the power_level by that number
 
+class SuperHero
+  attr_accessor :name, :super_power, :age
+  def initialize(name, super_power, age)
+    @name = name
+    @super_power = super_power
+    @age = age
+
+    @@arch_nemesis = "The Syntax Error"
+    @@power_level = 100
+    @@energy_level = 50
+  end
+
+  def say_name
+    puts name
+  end
+
+  def maximize_energy
+    @@energy_level = 1000
+  end
+
+  def gain_power(number)
+    puts @@power_level * number
+  end
+end
+
 # - Create 2 instances of your SuperHero class
 
+super_person_hero = SuperHero.new('Super Person', 'invisibility', 33)
+puts super_person_hero.say_name
+puts super_person_hero.gain_power(3)
+
+
 
 # Reflection
 # What parts were most difficult about this exerise?
+# Creating classes was the most difficult for me.
 
 # What parts felt most comfortable to you?
+# Creating arrays felt the most comfortable.
 
 # What skills do you need to continue to practice before starting Mod 1?
-
+# I need to continue practicing classes and memorizing the syntax.
```

Comment:
Yeah, with time and practice, it will become automatic to you. Don't push too hard on memorization - your problem solving skills matter most and you can always google for the syntax if needed! You are doing great.